cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -899,6 +928,10 @@ struct preprocessed_table {
   std::vector<rmm::device_buffer> _null_buffers;
 };
 
+/**
+ * @brief Comparator for performing equality comparisons between two rows of the same table.
+ *
```

Comment:
Does doxygen style require this extra line after the brief even when there are no additional lines afterwards, or is that something that you are doing? It seems like an unnecessary extra line to me, but maybe it's just what we need to do?

Reply:
doxygen does not require it. The vscode doxygen extension inserts this line by default.

Reply:
I see. I think I saw a few other places where there were param/return docs after the brief, and there was no space in between. It would be nice to make those consistent, but if doxygen doesn't require it then I wouldn't waste time on that (except when adding new lines) until after these PRs are all set.

Reply:
AFAIK, new lines are for readability. I would prefer to leave it as the doxygen extension generates.
it does not affect generated docs.

I will create a single PR with other changes like removing dots at end of `@param`, order of https://github.com/rapidsai/cudf/blob/branch-22.06/cpp/docs/DOCUMENTATION.md#function-parameters , `//!<` to `///<`.

do you suggest to add "compulsory newline after `@brief`"?